webapi/src/main/scala/org/knora/webapi/messages/v2/responder/resourcemessages/ResourceMessagesV2.scala

```diff
@@ -674,7 +674,7 @@ object CreateResourceRequestV2 {
   ] =
     ZIO.serviceWithZIO[StringFormatter] { implicit stringFormatter =>
       val validationFun: (String, => Nothing) => String =
-        (s, errorFun) => Iri.toSparqlEncodedString(s).getOrElse(errorFun)
+        (s, errorFun) => s // Iri.toSparqlEncodedString(s).getOrElse(errorFun)
```

Comment:
Should this commented out code be removed?

Reply:
Yes, absolutely, one second.

Reply:
Removed, inlined where necessary, left the original untouched.